kasa/cli.py

```diff
@@ -81,6 +81,36 @@ def cli(ctx, host, alias, target, debug, bulb, plug, strip):
         ctx.invoke(state)
 
 
+@cli.group()
+@pass_dev
+def wifi(dev):
+    """Commands to control wifi settings."""
+    pass
```

Comment:
`pass` isn't needed with the doc-string.

Reply:
Indeed, I added the doc-string afterwards and forgot to remove that :+1: